debian-adapter/pom.xml

```diff
@@ -104,25 +104,13 @@ SOFTWARE.
     <dependency>
       <groupId>com.adobe.testing</groupId>
       <artifactId>s3mock</artifactId>
-      <version>2.4.9</version>
+      <version>3.4.0</version>
```

Comment:
We should create a property for `s3mock`'s version in the parent pom

Reply:
@dgarus , I can make local variable, like for `jettyVersion`. For some reason `jettyVersion` is not inherited from parent pom - without local declaration maven does not proceed. Also IDE Idea 2022 & 2023 with WSL2 makes projects with submodules hardly usable due to several bugs, so I open every module as separate project and need local declaration.

Reply:
I don't like an approach when you have to change several places to update one artifact version.
Look at the `guava.version` property as an aexample.

Reply:
@dgarus , ok, I'll check my environment, maybe also something broken in it.